### PR TITLE
Merge branch 'upstream/master'

### PR DESCRIPTION
**Pull request by DmitriyMolch**
https://cerenetwork.atlassian.net/browse/CBI-2284

### CODE REVIEW

File: .github/workflows/dev.yaml [hunk 1]
```diff
@@ -24,7 +24,7 @@ jobs:
       - name: Install packages
         run: yarn install
       - name: Build
-        run: yarn build
+        run: yarn build:www
```
Comment (AndreiNavoichyk) [inline .github/workflows/dev.yaml:27]: Please, apply this fix to all envs

File: packages/apps-config/src/endpoints/production.ts [hunk 2]
```diff
@@ -46,7 +46,7 @@ export const prodChains: EndpointOption[] = [
     text: 'Cere Mainnet',
     providers: {
       'Cere Network': 'wss://ext-devs-node-3.cluster-3.cere.network:9945',
-      'Republic Crypto': 'wss://cere-archive.centrifuge.labs.republiccrypto.com:444',
+      'Republic Crypto': 'wss://ext-devs-node-3.cluster-3.cere.network:9945#republic',
```
Comment (AndreiNavoichyk) [inline packages/apps-config/src/endpoints/production.ts:49]: Please, revert value for `Cere Network` and `Republic Crypto` back

Comment (DmitriyMolch) [reply packages/apps-config/src/endpoints/production.ts]: fixed

Comment (DmitriyMolch) [reply packages/apps-config/src/endpoints/production.ts]: fixed

**Comment by AndreiNavoichyk** [comment]
@DmitriyMolch 
Please, update Cere's custom types. The latest you can find [here](https://www.notion.so/cere/Restore-the-network-to-the-current-state-7cd7761d5dfb45f1992979ddb1aed99b#33ed906efa1e4fea8a6ae2279bd10017).

**Review by AndreiNavoichyk — COMMENTED** [review]
We should have original commits in the history. Now we don't. Please, talk with @MaksimR on how to achieve this.

**Comment by DmitriyMolch** [comment]
Closed it as it contains squashed history.

**Pull request closed without merging**